commands/games/battleship.py

```
@@ -0,0 +1,404 @@
+            rows.append(row_str)
+        header = "  " + COL_LABELS
+        return header + "\n" + "\n".join(rows)
+
+    def _format_battle_embed(self, locale: str) -> discord.Embed:
+        """Build the battle phase embed showing both boards."""
+        p1_name = self.player1.display_name
+        p2_name = self.player2.display_name if self.player2 != "tanjun" else "Tanjun"  # type: ignore[union-attr]
+
+        # Player 1: own board and view of opponent
+        p1_own = self._board_to_str(self.board1)
+        p1_view = self._board_to_str(self.view1)
```

Flake8 (7.3.0)

[error] local variable 'p1_view' is assigned to but never used

(F841)